examples/fork-variable-access/tests/fork_variable_access_test.bal

```diff
@@ -44,14 +45,14 @@ function testFunc() {
     } else {
         // The output is in random order
         foreach var x in outputs {
-            string value = string.convert(x);
-            if (value.equalsIgnoreCase("[value type variables] before fork: value of integer variable is [100] value of string variable is [WSO2]")) {
+            string value = x.toString();
+            if (stringutils:equalsIgnoreCase(value, "[value type variables] before fork: value of integer variable is [100] value of string variable is [WSO2]")) {
```

Comment:
I guess we can use `equalsIgnoreCaseAscii` from `lang.string` here. Isn't it working here ?

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/544/commits/535ae373dc7ed68adcd520bb0b467abe1b796b13